content/resources/Reference/Framework_versions.md

```diff
@@ -56,3 +56,75 @@ We have two flavors of releases:
 - [ES6 packages](/divingDeeper/developWithBjs/treeShaking)
 
 When developing please make sure to pick the one that fits your architecture. We recommend using the ES6 packages which allow your to reduce the final release's size using tree shaking.
+
+# CDN Current Versions

```

Comment:
This should be a 2nd level header (`##`) and not a main header. The following titles should follow as well

Reply:
Done